examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -1605,6 +1653,11 @@ int findGoldilocksModels(int argc, char* argv[]) {
   int iter_start = FLAGS_iter_start;
   int max_outer_iter = FLAGS_max_outer_iter;
   double stopping_threshold = 1e-4;
+
+  // beta_momentum = 0 means we only use gradient at current iter.
+  // Momentum can give you faster convergence. And get out of a local minimum
+  // caused by step size. See: https://distill.pub/2017/momentum/ WARNING:
+  // beta_momentum is not used in newton's method
```

Comment:
This already exist in line 92. Should remove this

Reply:
removed